front/lib/api/assistant/messages.ts

```diff
@@ -671,21 +675,42 @@ export async function batchRenderMessages<V extends RenderMessageVariant>(
     ConversationError
   >
 > {
-  const [userMessages, agentMessagesRes, contentFragments] = await Promise.all([
-    batchRenderUserMessages(auth, messages),
-    batchRenderAgentMessages(auth, messages, viewType),
-    batchRenderContentFragment(auth, conversation.sId, messages),
-  ]);
+  const [userMessages, agentMessagesRes, contentFragments, reactionsRes] =
+    await Promise.all([
+      batchRenderUserMessages(auth, messages),
+      batchRenderAgentMessages(auth, messages, viewType),
+      batchRenderContentFragment(auth, conversation.sId, messages),
+      getMessageReactions(auth, conversation.toJSON()),
```

Comment:
When I initially implemented reactions we decided to have them loaded from a separate route and then loaded with a `useConversationReactions` hook. 

Here is the PR:  https://github.com/dust-tt/dust/pull/1877/changes#diff-579ddf4e0a9be109132c5207e735ddf366090325127db84c60f5b24c53a854f1


I still think this is true and reactions should be loaded async. So I would keep the former path that we initially implemented, we still have the route and I guess some code could be reused from my old PR 🙏🏻